wire-ios-data-model/Source/Model/AccountStore.swift

```diff
@@ -41,7 +42,7 @@ public final class AccountStore: NSObject {
     public required init(root: URL) {
         directory = root.appendingPathComponent(AccountStore.directoryName)
         super.init()
-        try! FileManager.default.createAndProtectDirectory(at: directory)
+        try! fileManager.createAndProtectDirectory(at: directory)
```

Comment:
we should avoid force unwrap when possible, we could make it safer by making the init throwable and a do a try catch at callsite

Reply:
I think fixing this is outside the scope of the PR because it won't just be a matter of logging an error at the call site. 

Essentially if this fails in the best case the app is unusable because we won't be able to login etc -  we would not be able to recover from this failure.

I think you are right but the only way to handle this correctly I think would be to display a UI at app startup that the app is not usable. Making this work would require big changes.

Reply:
I agree that if it involves too much changes this should not be done in this PR but if the changes don't have too much impact or require little work, I think we should fix them when we can.

Reply:
Absolutely agree!